src/libexpr/print-options.hh

```diff
@@ -5,6 +5,7 @@
  */
 
 #include <limits>
+#include <stddef.h>
```

Comment:
```suggestion
#include <cstddef>
```

Reply:
I wasn't sure, which one was preferred, because it seemed to be not handled consistently so far.